### ISSUE TITLE
riak_kv_entropy_info dialyzer error

### ISSUE THREAD

**Issue by reiddraper** [basho/riak_kv#902]
@Vagabond and I couldn't figure out how to fix this one, since there is domain expertise required:

```
riak_kv_entropy_info.erl:165: The call riak_kv_entropy_info:handle_index_info(Cmd::{'tree_built',_} | {'exchange_complete',_,_,_},Key::{_,_},Info::any()) breaks the contract (term(),index(),index_info()) -> index_info()
```

**Comment by kellymclaughlin** [comment]
Is the question you're trying to answer why is the second parameter to `handle_index_info` even needed since it is not used in either function clause?

**Comment by reiddraper** [comment]
Oops, looks like I just misread the function arguments. I think I was seeing the second element of the _tuple_ instead of the second argument to the function.

**Comment by reiddraper** [comment]
Fixed in #904, if it gets merged.

**Comment by reiddraper** [comment]
#904 was merged, closing.
